<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_widget</class>
 <widget class="QWidget" name="main_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/image/big_background.png);
</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>611</width>
     <height>571</height>
    </rect>
   </property>
   <widget class="QPushButton" name="btn_jc">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>250</width>
      <height>55</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>四则运算器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_hanshu">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>250</width>
      <height>55</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>函数运算器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_trigo">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>250</width>
      <height>55</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>三角函数运算器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_jzzh">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>300</y>
      <width>250</width>
      <height>55</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>进制转换器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_jzys">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>390</y>
      <width>250</width>
      <height>55</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>进制运算器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_week">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>480</y>
      <width>250</width>
      <height>55</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>今天是星期几？</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
